### ISSUE TITLE
Some Way to Track the Progress on a Delete Request

### ISSUE THREAD

**Issue by ehatch90** [kairosdb/kairosdb#367]
We have some bad data that I am trying to delete. The delete request has taken a very long time. I think that some way of monitoring whether that delete is still doing anything would be helpful. The command hasn't returned so I am assuming it's still working. Printing to the log every once in a while could be helpful in affirming that it's still working. I don't think this would be necessary for all deletes but maybe for deletes taking longer than X period of time (x being chosen in the properties file, with a default if not set).

**Comment by brianhks** [comment]
Yes, a delete could report to the log every 3 week boundary that it hits during the delete.  Then you could see it progress through the data without filling the logs with too much cruft.
